typhon/cloudmask/aster.py

```diff
@@ -224,7 +236,7 @@ def get_reflectance(self, channel):
 
         return (np.pi * self.get_radiance(channel) * distance**2 /
                 E_sun[self.channels.index(channel)] /
-                np.cos(np.radians(sunzenith))
+                np.cos(np.radians(self.sunzenith))
```

Comment:
You may use ``np.deg2rad`` to be more verbose, but that's up to you as both functions are equivalent 😉